Podfile

```diff
@@ -13,4 +13,5 @@ pod 'CVCalendar', '~> 1.2.8'
 pod 'SimpleCircleProgressView', '~> 1.0.3'
 pod 'HorizontalProgressView', '~> 0.9.5'
 pod 'PickerSwift', '~> 0.9.3'
-pod 'DoneToolbarSwift', '~> 1.0.0'
\ No newline at end of file
+pod 'DoneToolbarSwift', '~> 1.0.0'
+pod 'GoogleMaps’, ‘~> 1.12.3'
```

Comment:
sanitise the quotes (don't use the default editor :P). You have a " ´ " instead of a single quote "  '  ". Pods give you a warning